### ISSUE TITLE
map of pictures taken

### ISSUE THREAD

**Issue by vict0rsch** [cc-ai/climatecollection#1]
v2

**Comment by Vahe987** [comment]
what is this, can you be more specific? if it is something old, close this please @vict0rsch

**Comment by vict0rsch** [comment]
This is one of the ideas discussed, for later maybe: provide a map of where users uploaded pictures from (without the pictures themselves to preserve privacy, and maybe not complete address but city)

If you don't want to use issues a feature-suggestion tool let's use something else @Vahe987

**Comment by Vahe987** [comment]
no, it's fine, I was just going trough issues to see which one should be closed, which one should be our focus right now, etc. probably they are better tools to discuss features, but no need to reorganize everything :) 
however all suggestions are welcomed ;)
